ftw/notification/base/browser/views.py

```diff
@@ -132,7 +133,8 @@ def users(self):
                 if member is None:
                     continue
 
-                user = dict(title=t.title,
+                title = t.title if ITitledTokenizedTerm.providedBy(t) else t.value
```

Comment:
Ok, I see. Since this is a vocabulary, we have to suspect that the `t`s are some kind of terms, thus have a `value`.
If this is not the case there is a problem with the vocabulary.
I think we can give it a shot :wink: